buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -554,11 +554,11 @@ class BeamModulePlugin implements Plugin<Project> {
     def log4j2_version = "2.20.0"
     def nemo_version = "0.1"
     // Try to keep netty_version consistent with the netty version in grpc_bom (includes grpc_netty) in google_cloud_platform_libraries_bom
-    def netty_version = "4.1.87.Final"
+    def netty_version = "4.1.96.Final"
```

Comment:
Any reason for this particular bump?

I see 4.1.87 is still the one declared by the BOM

```
[INFO] +- io.grpc:grpc-netty:jar:1.56.1:compile
[INFO] |  +- io.netty:netty-codec-http2:jar:4.1.87.Final:compile
```

Reply:
reverted